lib/utils.ts

```diff
@@ -0,0 +1,9 @@
+export function normalizeURL(
+  url: string,
+  { baseURL = "" }: { baseURL: string }
```

Comment:
Do we need a default value? If we're only using this when we have `baseURL` I prefer a stricter API.

Reply:
Good call! Updated.